invenio_rdm_records/services/generators.py

```diff
@@ -135,6 +139,58 @@ def _condition(self, record=None, **kwargs):
             return False
 
 
+class IfRecordDeleted(Generator):
```

Comment:
@max-moser added https://github.com/inveniosoftware/invenio-communities/commit/31945c6954780fc32f56f1cd16c3163ca2bb8216#diff-2476adbb5df07ac52ab8b38741dacd9f144d1ee77dbcef244085581885d6c530R148 and it seems similar so we need probably to align if possible these 2 approaches

Reply:
I didn't use it intentionally, he created one also for records, see https://github.com/inveniosoftware/invenio-rdm-records/blob/master/invenio_rdm_records/services/permissions.py#L174
this was used for edit, and it has slightly different purpose, I didn't want to mix them. Maybe I should rename or Max'es, do you have any idea what would be a good naming ?